tests/test_converter.py

```diff
@@ -1,24 +1,24 @@
-from nose.tools import assert_equal
+import unittest
```

Comment:
I prefer to use pytest for everything, and not use a mix of unittest and pytest. Could you pytest'ify all the tests? 

E.g. many tests can be parametrized. And a lot of tests don't need to be grouped in a class anymore.

I'm ok with a proper rewrite/cleanup of the tests.